### ISSUE TITLE
Accessibility meta page

### ISSUE THREAD

**Issue by dngray** [privacyguides/privacyguides.org#2511]
### Bug description

We should have an accessibility meta page that mentions things like this:

> https://webaim.org/resources/contrastchecker/

### Before submitting

- [X] I am reporting something that is broken on the website, not making a suggestion.
- [X] I agree to the [Community Code of Conduct](https://www.privacyguides.org/coc).
